user/src/main/resources/application.yml

```
@@ -61,3 +61,8 @@ management:
       show-details: always
     prometheus:
       access: unrestricted
+
+
+kakao:
+  api:
+    key: ${KAKAO_MAP_KEY}
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)